### PR TITLE
CIV-7615 Translate Uncommon DQ fields to CUI

### PR DESCRIPTION
**Pull request by laperla11**
### JIRA link ###
https://tools.hmcts.net/jira/browse/CIV-7615

### Change description ###
Translate CCD data to CUI DQ model for uncommon DQ screens for Small claims and Fast-track claims

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: src/main/services/translation/convertToCUI/convertToCUIDQ/convertToCUIDirectionQuestionnaire.ts [hunk 1]
```diff
@@ -0,0 +1,63 @@
+import {CCDClaim} from 'common/models/civilClaimResponse';
```
Comment (laperla11) [inline src/main/services/translation/convertToCUI/convertToCUIDQ/convertToCUIDirectionQuestionnaire.ts:1]: CIV-7615 changes

File: src/main/services/translation/convertToCUI/convertToCUIDQ/covertToCUIDQExperts.ts [hunk 2]
```diff
@@ -0,0 +1,70 @@
+import {YesNoNotReceived} from 'common/form/models/yesNo';
```
Comment (laperla11) [inline src/main/services/translation/convertToCUI/convertToCUIDQ/covertToCUIDQExperts.ts:1]: CIV-7615 changes

File: src/main/services/translation/convertToCUI/cuiTranslation.ts [hunk 3]
```diff
@@ -16,6 +16,7 @@ export const translateCCDCaseDataToCUIModel = (ccdClaim: CCDClaim): Claim => {
   claim.mediation = toCUIMediation(ccdClaim?.respondent1LiPResponse?.respondent1MediationLiPResponse);
   claim.statementOfMeans = toCUIStatementOfMeans(ccdClaim);
   claim.claimBilingualLanguagePreference = toCUIClaimBilingualLangPreference(ccdClaim?.respondent1LiPResponse?.respondent1ResponseLanguage);
+  claim.directionQuestionnaire = toCUIDirectionQuestionnaire(ccdClaim);
```
Comment (laperla11) [inline src/main/services/translation/convertToCUI/cuiTranslation.ts:19]: CIV-7615 changes

File: src/test/unit/services/translation/convertToCUI/covertToCUIDQ/convertToCUIDirectionQuestionnaire.test.ts [hunk 4]
```diff
@@ -0,0 +1,151 @@
+import {
```
Comment (laperla11) [inline src/test/unit/services/translation/convertToCUI/covertToCUIDQ/convertToCUIDirectionQuestionnaire.test.ts:1]: CIV-7615 changes

File: src/test/unit/services/translation/convertToCUI/covertToCUIDQ/covertToCUIDQExperts.test.ts [hunk 5]
```diff
@@ -0,0 +1,165 @@
+import {
```
Comment (laperla11) [inline src/test/unit/services/translation/convertToCUI/covertToCUIDQ/covertToCUIDQExperts.test.ts:1]: CIV-7615 changes

**Review by LeonardoPalmeiro — APPROVED** [review]

**Review by TatsyMaslova — APPROVED** [review]

**Review by portizdev — DISMISSED** [review]

**Review by GorilaNaranja — APPROVED** [review]

**Review by neeta-hmcts — APPROVED** [review]

**Review by hmcts-version1-domini — APPROVED** [review]

**Review by portizdev — APPROVED** [review]

**Review by rajakm — APPROVED** [review]

**Pull request merged**